frameworks/kafka/tests/test_availability.py

```diff
@@ -40,7 +40,6 @@ def teardown_module(module):
     sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
 
 
-@pytest.mark.paegun
 @pytest.mark.availability
```

Comment:
@paegun the test below fails due to the use of `pods restart` instead of `pod restart`. An additional point is that the `retries` variable is not decremented when running `topic producer_tests test`. Would make sense to rather use the `retrying` module here if this is what we want? This would allow us to, for example, raise an exception in the failure case.

Reply:
Yes, it would be better to use `retrying`, but IMO we should apply this throughout, not be inconsistent.